packages/markdown-docx/package.json

```diff
@@ -26,8 +26,8 @@
     "lint": "eslint .",
     "postlint": "npm run licchk",
     "licchk": "license-check",
-    "test": "jest --timeOut=10000",
-    "test:cov": "npm run lint && jest --timeOut=10000 --coverage",
+    "test": "mocha -t 10000",
+    "test:cov": "npm run lint && nyc npm run test",
```

Comment:
Since you are not using `jest` now, you should remove the package too and make sure you reflect that in `package-lock.json`.

Reply:
@algomaster99 
Running `lerna bootstrap` has created no changes in `package-lock.json`. Should I push the changes made?

Reply:
Did it not delete the jest framework?

Reply:
I tried finding `jest` with `ctrl+f` but couldn't find any in `package-lock.json`. So I think maybe it would not have been created in first place.